### ISSUE TITLE
Unexpected tokens in vcd that loads in `gtkwave`

### ISSUE THREAD

**Issue by rroohhh** [ekiwi/wellen#28]
Running the `load_signals` example on the [attached vcd](https://github.com/user-attachments/files/17342069/out.zip)  (zipped to be able to upload here) file yields the following output:
```
It took 59.343885ms to load the header of /data/study/master/thesis/work/toplevel/out.vcd
[00:00:00] ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ 0 B (0.000%)                                                                                  thread 'thread '<unnamed>' panicked at wellen/src/wavemem.rs:490:25:
index out of bounds: the len is 7993 but the index is 11156603771105
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `jm` and `0km` (1067 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `E^` and `0F^` (81 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `$D` and `0%D` (833 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `aJ` and `0bJ` (155 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `9s` and `0:s` (197 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `vX` and `0wX` (943 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `!/` and `0"/` (705 lines after header)
thread '<unnamed>' panicked at wellen/src/vcd.rs:1171:33:
Unexpected tokens: `j5` and `0k5` (47 lines after header)
[00:00:00] ███████████████████████░░░░░░░░░░░░░░░░░ 336.33 kB (57.560%)
```
while `gtkwave` loads it without problems. 
If you need a smaller file I can probably produce one, let me know if that would be useful.

**Comment by ekiwi** [comment]
I am looking into this now (finally!).

This is what `wellen` is currently stumbling over:
```
$var wire 0 H read_idx $end
...
b H
```
I.e., we currently do not deal well with 0-bit signals

**Comment by ekiwi** [comment]
This should be fixed in wellen `0.13.3`.
Please re-open this issue if the problem persists.
